### ISSUE TITLE
Puppeteer Error

### ISSUE THREAD

**Issue by RubenDanielMontemayor** [kylon/Sharedown#8]
My university uses DUO for our logins as a 2FA. Somehow I think this interferes with the way Sharedown works, even with setting the timeout to 0 seconds (essentially no timeout). I get an error as described in the image: 
![image](https://user-images.githubusercontent.com/89608049/133580112-f36b0a57-c957-4678-92c5-c69a34f205ba.png)

**Comment by kylon** [comment]
test version

[Sharedown-t1.zip](https://github.com/kylon/Sharedown/files/7176506/Sharedown-t1.zip)

you need [node.js](https://nodejs.org/dist/v16.9.1/node-v16.9.1-x86.msi)

instructions:

unpack to a folder
open cmd in that folder
type `npm i`
when ready, type `npm start` to run sharedown

**Comment by RubenDanielMontemayor** [comment]
The test version attempts to open the video file in the Chromium browser, but it ends up freezing and nothing happens: 
![image](https://user-images.githubusercontent.com/89608049/133667797-00b54d9f-82b7-4bcc-bdb5-7a22701c4468.png)

Also, the Sharedown app itself get stuck in some sort of loading screen: 
![image](https://user-images.githubusercontent.com/89608049/133667873-ee627671-8314-449f-9741-a8b48ecbef19.png)

Here's a copy of what the installation was like through the command prompt:
[Sharedown Command Prompt Output.txt](https://github.com/kylon/Sharedown/files/7180262/Sharedown.Command.Prompt.Output.txt)

**Comment by RubenDanielMontemayor** [comment]
My mistake, I had installed the LTS version of Node.js onto my computer, but I see now that you requested the newer Current version. After doing a whole installation (including Chocolatey and other programs), and repairing Node.js (it crashed the first time I tried to install it), I finally made some progress on Sharedown.

This time, after loading the video player into the Chromium browser, it closed the browser, and then attempted to download the video. It barely started the process, and then gave me an error:
![image](https://user-images.githubusercontent.com/89608049/133673489-1caf6c4e-864a-48bf-8471-4f13b9b05a23.png)

I'm going to try the ffmpeg version and see how it fares.
